de.fu_berlin.inf.dpp/src/de/fu_berlin/inf/dpp/EclipsePluginLifecycle.java

```diff
@@ -55,8 +56,10 @@ private EclipsePluginLifecycle(Saros saros) {
 
   @Override
   protected void initializeContext(final ContainerContext containerContext) {
-    FeedbackPreferences.setPreferences(
-        containerContext.getComponent(org.osgi.service.prefs.Preferences.class));
+    if (FeedbackManager.isFeedbackFeatureRequired()) {
+      FeedbackPreferences.setPreferences(
```

Comment:
If you look at the class there are several get methods. And if you call them without calling this method you are triggering RuntimeExceptions.  This call is harmless.

Reply:
Ok, I will remove this check